diagnostics/README.md

```diff
@@ -68,40 +58,19 @@ For example usage, take a look at the ```model_test/verify_model.py``` in the di
 
 ## Protobuf visualization
 
-A `ngtf_graph_viewer` script can convert a protobuf (`pb` or `pbtxt`) into a 
-dot file or a TB log, which can be viewed using TB. If the input is a `pbtxt`, 
-it can also sanitize node names to remove underscores from the front of node 
-names. It can also prepend strings in front of certain node names, a feature 
-which can be used to append encapsulate information for clustering nodes 
-together. (Tested on Python 2 TF-1.9; however, it should work with Python 3 
-and other versions of TF.)
-
-Run the following for detailed help:
-
-    python ngtf_graph_viewer.py -h
-
-
-### Examples for command-line use
-
-* `pbtxt` to TB: ```python ngtf_graph_viewer.py pbtxtfile.pbtxt ./vis```
-* `pbtxt` to dot: ```python ngtf_graph_viewer.py -v 0 pbtxtfile.pbtxt ./vis```
-* `pb` to TB: ```python ngtf_graph_viewer.py -b pbtxtfile.pb ./vis```
-* `pb` to dot: ```python ngtf_graph_viewer.py -b -v 0 pbtxtfile.pb ./vis```
-* `pbtxt` to TB (after prepending cluster information): ```python ngtf_graph_viewer.py -c nodemap.pkl pbtxtfile.pbtxt ./vis``` 
-
-
-### Some other use cases
+The [tf2ngraph] script can be used to convert tensorflow graph to an ngraph enabled graph
```

Comment:
This is not very user friendly. Explain the use case in more detailed and provide better example.